quiz.js

```diff
@@ -72,3 +74,46 @@ const questions = [
       <p>${contentText}</p>
     `;
   }
+
+  //INFORMATION PANEL
+let step = 'step1';
+
+const step1 = document.getElementById('step1');
+const step2 = document.getElementById('step2');
+const step3 = document.getElementById('step3');
+const step4 = document.getElementById('step4');
+
+function next() {
```

Comment:
Ta funkcja jest poprawna, działa, ale mam do niej kilka uwag i sugestii :-)

Pierwsze co rzuca mi się w oczy to fakt że w tym pliku istnieje już funkcja która się uruchamia w momencie kiedy przycisk "dalej" jest naciśnięty, i jest ona podłączona do tego przycisku przez addEventListener w pliku JS, nie przez onClick w pliku HTML. Warto by było zachować konsystencję i podłączyć twoją funkcję w ten sam sposób, a w ogóle jeszcze lepiej byłoby użyć istniejącej funkcji. Tam jest już zmienna currentQuestion która, jak rozumiem, jest analogiczna do twojej zmiennej step. Nie ma po co tego duplikować :-). Zachowałbym twoją funkcję do podświetlania "stepu" jako osobną, ale wywołał ją z istniejącej funkcji showNextQuestion zamiast podpinać ją z boku, oraz skorzystałbym z istniejącej zmiennej currentQuestion zamiast tworzyć nową zmienną step.

Ponadto, przy obecnej implementacji, aby dodać kolejny krok, trzeba zrobić strasznie dużo zmian. Gdybym chciał mieć teraz 50 kroków zamiast 5 to musiałbym się solidnie narobić. Można jednak przerobić tę funkcję tak, żeby obsługiwała dowolną ilość kroków, np. tak:
- wybierz wszystkie elementy od stepów na raz, np. document.querySelectorAll('.progress-step')
- usuń niepotrzebne klasy ze wszystkich stepów
- na podstawie zmiennej currentQuestion wybierz element w sparametryzowany sposób, np. document.getElementById(`step${currentQuestion}`)
- dodaj do tego elementu potrzebne klasy

Dodatkowo warunek końca masz za darmo już obsłużony w istniejącej funkcji showNextQuestion :-)